### PR TITLE
saving player names in map to reduce SQL query

### PR DESCRIPTION
**Pull request by XeAri**

### CODE REVIEW

File: src/servers/sapphire_zone/ServerMgr.h [hunk 1]
```diff
@@ -59,6 +59,7 @@ class ServerMgr
 
   std::map< uint32_t, SessionPtr > m_sessionMapById;
   std::map< std::string, SessionPtr > m_sessionMapByName;
+  std::map< uint32_t, std::string > m_payerNameMap;
```
Comment (NotAdam) [inline src/servers/sapphire_zone/ServerMgr.h]: `m_playerNameMap`

Comment (SapphireMordred) [reply src/servers/sapphire_zone/ServerMgr.h]: Name should also indicate what is mapped to the name. See the other maps

**Comment by NotAdam** [comment]
while this kinda alleviates the problem, it also means if the server is running for a long time and someone changes their name, it won't update on the house

File: src/servers/sapphire_zone/ServerMgr.cpp [hunk 2]
```diff
@@ -380,11 +380,19 @@ std::string Core::ServerMgr::getPlayerNameFromDb( uint32_t playerId )
 
   std::string playerName = res->getString( 1 );
 
-  m_payerNameMap[ playerId ] = playerName;
+  m_payerNameMapById[ playerId ] = playerName;
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/ServerMgr.cpp]: Good, except i don't think you are looking for a payer name here ;)

**Pull request merged**